.insomnia/Request/req_c3a247ec8a8c48d8a0ef3cf39cdfb735.yml

```diff
@@ -0,0 +1,55 @@
+_id: req_c3a247ec8a8c48d8a0ef3cf39cdfb735
+type: Request
+parentId: fld_eeb62a531f8441478a6d2f536021400d
+modified: 1658855308155
+created: 1658848365376
+url: "{{ _.base_url }}/api/v1/reviews"
+name: Reviews
+description: >-
+  This endpoint is used to create reviews given certain parameters such as
+  review title, description and metadata attributes.
+
+
+  Note that parameters are passed as a URL-encoded form.
+method: POST
+body:
+  mimeType: application/x-www-form-urlencoded
+  params:
```

Comment:
There are more parameters to this request, more metadata and more importantly team id (if user can create team reviews).

Reply:
What exactly are the other parameters?

Reply:
Basically everything in the metadata object. You can check the models.

Reply:
I believe I've covered everything?